pkgs/by-name/re/resnap/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenvNoCC,
+  writeShellApplication,
+  ffmpeg,
+  feh,
+  imagemagick_light,
+  lz4,
+  fetchFromGitHub,
+}:
+let
+  name = "reSnap";
```

Comment:
```suggestion
  pname = "resnap";
```

Reply:
writeShellApplication has no `pname` attribute. You'd want to do this with `stdenv.mkDerivation`